src/backend/mysql.rs

```diff
@@ -30,7 +30,7 @@ impl SqlGenerator for MySql {
     }
 
     fn alter_table(name: &str) -> String {
-        format!("ALTER TABLE \"{}\"", name)
+        format!("ALTER TABLE {}", name)
```

Comment:
I'm not sure I understand the validity of this change. Setting the database name in quotes is generally good practise, I've found

Reply:
With the quotes, the generated query fails.

Reply:
Interesting. I would have thought that `mysql` would be fine with setting the database name in quotations.

Pinging @rippinrobr who wrote the `mysql` module for this.

Reply:
Hmmm...okay so looking into this it feels like backticks would generally be preferred when generating MySQL migrations, right?

I would suggest the following:

1. Force push that last commit where you take out the changes done to `src/backend/mysql.rs`
2. Make another commit where you change all the `"` in that file to `. Then I'll rebase that so that the git history looks prettier :)
3. You also need to adjust the tests to the new expected output because right now those are failing.

Still kinda curious what @rippinrobr thinks about this, since I know they've been using this module. But we don't necessarily have to wait with a merge for that.

Reply:
When I use `"` in both mariadb and mysql it works fine in both but I just looked at a database back up and sure enough, all of the tables have ` around them.

INSERT INTO `appearances` VALUES
LOCK TABLES `appearances` WRITE;
DROP TABLE IF EXISTS `awardsmanagers`;

So I would say back ticks are the preferred way of quoting table names.

Reply:
@spacekookie - everything looks to be good now :)

Reply:
@rippinrobr - FYI, the double quotes only work when ANSI mode is enabled for MySQL:

https://dev.mysql.com/doc/refman/5.5/en/sql-mode.html#sqlmode_ansi_quotes